### PR TITLE
chore: add AI Colors under the frontend/colors subcategory

### LINKED ISSUE #862: [Add] AIColors under Colors
### Site URL

https://aicolors.co/

### Description of the above provided site

- This website is one of the most awesome websites for generating validated color palettes.
- Users can generate a color palette based on a text prompt.
- Each color palette is visually displayed on a user interface design, which is what makes it game-changing.
- Additionally, users can save these palettes and explore the trending ones.

### Category

Frontend

### Sub Category

Colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
## Fixes Issue

Closes #862 

## Screenshots

image

### CODE REVIEW

File: database/frontend/colors.json [hunk 1]
```diff
@@ -93,22 +93,29 @@
...
+  {
     "name": "Khroma",
     "description": "This is an innovative color palette generator that simplifies the process of choosing harmonious colors for your design projects.",
     "url": "https://www.khroma.co/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+  {
+    "name": "AI Colors",
+    "description": "Generate stunning, validated color palettes generated based on text prompts and witness their practical application as they dynamically come to life on a generated user interface, allowing you to visualize and explore the impact of colors in real time.",
```
Comment (CBID2) [inline database/frontend/colors.json:116]: [Proofreading] 
```suggestion
    "description": "Generate stunning, validated color palettes  based on text prompts and witness their practical application as they dynamically come to life on a generated user interface, allowing you to visualize and explore the impact of colors in real-time.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Anmol-Baranwal! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by Anmol-Baranwal** [comment]
> Hi @Anmol-Baranwal! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

As you can see, I have checked it on Grammarly, and there were no errors. 

Anyways, I have made the changes.

image

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by ujjawaltyagii — APPROVED** [review]
LGTM !

File: database/frontend/colors.json (hunk 1 above)
Comment (rupali-codes) [reply database/frontend/colors.json]: ty Chirssy

**Pull request merged**